osf/utils/metadata/schemas/datacite.json

```diff
@@ -0,0 +1,607 @@
+{
```

Comment:
Can we add comments to a jsonschema? If so we may want to note that this came from an external source, just so we know and if we need to tweak it in the future!

Reply:
Sadly, comments aren't supported in JSON. However, this can apparently be hacked around with something like: 
```
{
	"__comment": "Schema copied from https://github.com/inveniosoftware/datacite/blob/v1.0.1/datacite/schemas/datacite-v4.0.json and should not be changed."
	...
}